params/config.go

```diff
@@ -100,6 +100,7 @@ type WakuV2Config struct {
 	EnableConfirmations bool
 
 	// A name->libp2p_addr map for Wakuv2 custom nodes
+	// Deprecated: simply unused
 	CustomNodes map[string]string
```

Comment:
Would it be possible to simply remove deprecated/unused fields?

Reply:
Yes! I think we should be able to remove the whole `ClusterConfig`, just move the `WakuNode`, `DiscV5BootstrapNodes` to `WakuConfig`.

I started to do it, but decided not to mess up this PR, just marked some stuff as deprecated to easily find later.